### PR TITLE
add tests of application.xml resource definitions, initially disabled

### PR DESCRIPTION
**Pull request by njr-11**
In preparation for adding resource definitions in deployment descriptors, add some tests.
We will start out with the resource definition annotations commented out and temporarily define equivalents annotatively, so that tests can easily be switched over to use the deployment descriptor resource definitions, once available.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

File: dev/com.ibm.ws.concurrent_fat_jakarta/test-applications/ConcurrencyTestApp/resources/META-INF/application.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nmittles) [inline dev/com.ibm.ws.concurrent_fat_jakarta/test-applications/ConcurrencyTestApp/resources/META-INF/application.xml:1]: Need copyright header?

**Review by nmittles — APPROVED** [review]

**Comment by njr-11** [comment]
#run-libby-bot
#build

File: dev/com.ibm.ws.concurrent_fat_jakarta/test-applications/ConcurrencyTestApp/resources/META-INF/application.xml (hunk 1 above)
Comment (njr-11) [reply dev/com.ibm.ws.concurrent_fat_jakarta/test-applications/ConcurrencyTestApp/resources/META-INF/application.xml:1]: Thanks for catching this. I added it under https://github.com/OpenLiberty/open-liberty/pull/19240/commits/fff368d5ed72a8e94a9bfdc024d82aa3b25620fc

**Pull request merged**